os/net/app-layer/coap/coap-callback-api.c

```diff
@@ -48,6 +48,7 @@
 #include "sys/cc.h"
 #include <stdlib.h>
 #include <string.h>
+#include <inttypes.h>
```

Comment:
The printf specifier for uint32_t (PRIu32) is used throughout the code, which are defined in inttypes.h. Since the header isn't included directly, it relies on inttypes.h being included indirectly, which isn't the case for this platform. That's why I've added the include. I can add this as separate PR if you want.

Reply:
Yes please